gen/calc.py

```diff
@@ -615,6 +616,7 @@ def validate_exhibitor_storage_master_discovery(master_discovery, exhibitor_stor
         'dcos_remove_dockercfg_enable': "false",
         'minuteman_min_named_ip': '11.0.0.0',
         'minuteman_max_named_ip': '11.255.255.255',
+        'minuteman_disable_bridge_netfilter': 'true',
```

Comment:
Should we be keeping the default value to `true`? I guess this should generally be fine for the current use cases we have. But we really need to document this better in the installation. In a setup like BSN this will wreak havoc !!